.github/workflows/curl-test.yml

```
@@ -0,0 +1,34 @@
+  workflow_dispatch:
+
+jobs:
+  curl-test:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      
+    - name: Curl Google
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:9:6: Double quote to prevent globbing and word splitting

(shellcheck)